src/lib.rs

```diff
@@ -118,10 +122,14 @@ impl Mnemonic {
 	}
 
 	/// Create a new [Mnemonic] in the specified language from the given entropy.
-	/// Entropy must be a multiple of 32 bits (4 bytes).
+	/// Entropy must be a multiple of 32 bits (4 bytes) and 128-256 bits in length.
 	pub fn from_entropy_in(language: Language, entropy: &[u8]) -> Result<Mnemonic, Error> {
 		if entropy.len() % 4 != 0 {
-			return Err(Error::BadEntropyBitCount(entropy.len() * 8));
+			return Err(Error::BadEntropyBitMultiple(entropy.len() * 8));
+		}
+
+		if (entropy.len() * 8) < 128 || (entropy.len() * 8) > 256 {
+			return Err(Error::BadEntropyBitLength(entropy.len() * 8));
```

Comment:
I think I'd prefer to use the same error type here. Right now when you get the `BadEntropyBitMultiple` error at first, you think using 32 or 64 bits is fine but then you'll just bump into the second error.

Reply:
@stevenroose make sense.  this has been updated.